Formula/tezos-accuser-009-PsFLoren.rb

```diff
@@ -32,6 +32,10 @@ class << self
   def make_deps
     ENV.deparallelize
     ENV["CARGO_HOME"]="./.cargo"
+    # Here is the workaround to use opam 2.0 because Tezos is currently not compatible with opam 2.1.0 and newer
+    system "curl", "-L", "https://github.com/ocaml/opam/releases/download/2.0.9/opam-2.0.9-x86_64-macos", "--create-dirs", "-o", "#{ENV["HOME"]}/.opam-bin/opam"
+    system "chmod", "+x", "#{ENV["HOME"]}/.opam-bin/opam"
+    ENV["PATH"]="#{ENV["HOME"]}/.opam-bin:#{ENV["PATH"]}"
```

Comment:
Two questions about this:
1. since it's used in so many formulae, can't it be extracted somewhere to avoid duplicating it?
2. AFAIU when building multiple formulae this will cause each of them to re-download and change permission to the opam binary, can we avoid to do that if the file already exists?

Reply:
> since it's used in so many formulae, can't it be extracted somewhere to avoid duplicating it?

ehh, `brew` doesn't like formulas that consist of multiple files :unamused: 

> AFAIU when building multiple formulae this will cause each of them to re-download and change permission to the opam binary, can we avoid to do that if the file already exists?

Build happens in an isolated environment, so we can't do much about it

Reply:
> doesn't like formulas that consist of multiple files
uh right, forgot about that problem :ok_hand: